tiphub-io/tiphub
# Fix Blockstack in Production + Meta

Closes #9.

### What This Does

* Adds a `/manifest.json` route to python that loads the static manifest.json file
* Adds configurable PUBLIC_PATH so that public paths are absolute, not relative
* Adjusts favicon and description

### Steps to Test

* Build production frontend and run backend with production configuration
* Go through blockstack auth
* Confirm it works correctly

<img width="470" alt="Screen Shot 2019-04-17 at 3 59 21 PM" src="https://user-images.githubusercontent.com/649992/56317011-c8af5d00-6129-11e9-9080-8c3c5527ca8a.png">
